### PR TITLE
fixing configuration(s) terms

### PR DESCRIPTION
**Pull request by interurban**
## Description
changing instances of Configurations and configurations to align product name : Pulumi ESC (Environments, Secrets and Configuration)
## Checklist:

- [ ] I have reviewed the [style guide](https://github.com/pulumi/pulumi-hugo/blob/master/STYLE-GUIDE.md).
- [ ] If blogging, I have reviewed the [blogging guide](https://github.com/pulumi/pulumi-hugo/blob/master/BLOGGING.md).
- [ ] I have manually confirmed that all new links work.
- [ ] I added aliases (i.e., redirects) for all filename changes.
- [ ] If making css changes, I rebuilt the bundle.

### CODE REVIEW

**Comment by elistaykova** [comment]
Hi @interurban and @aaronkao - can you also fix the ESC github??
1) the "ABOUT" section at the top right should list the proper name
2) the readme title in the description below should list the name properly

**Comment by elistaykova** [comment]
@interurban and @aaronkao - while you are at it, one more change please --> On the ESC product page, in the FAQ, can you please make the work "open source project" in this sentence
Pulumi ESC (Environments, Secrets and Configurations), is an open source project and managed service as part of

hyper link to here https://github.com/pulumi/esc 

Screenshot 2023-10-10 at 1 49 25 PM

File: themes/default/layouts/product/esc.html [hunk 1]
```diff
@@ -29,9 +29,9 @@ <h1 class="flex flex-col items-center">
...
+                    Today’s cloud environments access a multitude of configurations – including network settings, deployment options, API Keys, and other important secrets, such as database credentials –  from many different types of cloud infrastructure and SaaS services. Every team stores these configuration in different locations, from different secrets managers to plaintext files. This sprawl leads to configuration that are untraceable and lack access controls, leading to operational bottlenecks, outages linked to human error, and security breaches. Pulumi ESC enables you to centrally manage all configuration and secrets across your organization.
```
Comment (cnunciato) [inline themes/default/layouts/product/esc.html:34]:
```suggestion
                    Today’s cloud environments access a multitude of configurations – including network settings, deployment options, API Keys, and other important secrets like database credentials –  from many different types of cloud infrastructure and SaaS services. Every team stores configuration settings like these in different locations, from secrets managers to plaintext files. This sprawl leads to configuration that is uncontrolled and untraceable, leading to operational bottlenecks, outages linked to human error, and security breaches. Pulumi ESC enables you to centrally manage all configuration and secrets across your organization.
```

File: themes/default/layouts/product/esc.html [hunk 2]
```diff
@@ -40,8 +40,8 @@ <h3>Centralized environments, secrets, and configurations management for cloud a
...
+            <h3>Centralized environments, secrets, and configuration management for cloud applications and infrastructure</h3>
+            <p class="text-center">Today’s cloud environments access a multitude of configurations – including network settings, deployment options, API Keys, and other important secrets, such as database credentials –  from many different types of cloud infrastructure and SaaS services. Every team stores these configuration in different locations, from different secrets managers to plaintext files. This sprawl leads to configuration that are untraceable and lack access controls, leading to operational bottlenecks, outages linked to human error, and security breaches. Pulumi ESC enables you to centrally manage all configuration and secrets across your organization.</p>
```
Comment (cnunciato) [inline themes/default/layouts/product/esc.html:44]:
```suggestion
            <p class="text-center">Today’s cloud environments access a multitude of configurations – including network settings, deployment options, API Keys, and other important secrets like database credentials –  from many different types of cloud infrastructure and SaaS services. Every team stores configuration settings like these in different locations, from secrets managers to plaintext files. This sprawl leads to configuration that is uncontrolled and untraceable, leading to operational bottlenecks, outages linked to human error, and security breaches. Pulumi ESC enables you to centrally manage all configuration and secrets across your organization.</p>
```

Curious why this is duplicated, though; it'll be easy to change it in one place and forget to change it in the other.

File: themes/default/layouts/product/esc.html [hunk 3]
```diff
@@ -186,7 +186,7 @@ <h5>Use any secrets source</h5>
                         <h5>Access from anywhere</h5>
                     </div>
                     <div class="mt-6">
-                        <p>Consume configurations and secrets in an environment from any application or tool. You can use ESC for all your configuration and secrets needs independently of Pulumi’s core infrastructure as code offerings.  </p>
+                        <p>Consume configuration and secrets in an environment from any application or tool. You can use ESC for all your configuration and secrets needs independently of Pulumi’s core infrastructure as code offerings.  </p>
```
Comment (cnunciato) [inline themes/default/layouts/product/esc.html:189]:
```suggestion
                        <p>Consume configuration and secrets in any environment and from any application or tool. You can use Pulumi ESC for all your configuration and secrets needs independently of Pulumi’s core infrastructure as code offerings.</p>
```

File: themes/default/layouts/product/esc.html [hunk 4]
```diff
@@ -204,36 +204,36 @@ <h2>Frequently asked questions</h2>
         </div>
         <ul class="list-none p-0 mx-5 xl:ml-12" data-faq-type="faq-pricing">
             <li class="accordion-item text-2xl py-3">
-                {{ partial "accordian-header" (dict "text" "What is Pulumi Environments, Secrets and Configurations" "large_header" true) }}
+                {{ partial "accordian-header" (dict "text" "What is Pulumi Environments, Secrets and Configuration" "large_header" true) }}
```
Comment (cnunciato) [inline themes/default/layouts/product/esc.html:207]: Should always be either "Pulumi ESC" or "Pulumi ESC (and the words)":

```suggestion
                {{ partial "accordian-header" (dict "text" "What is Pulumi ESC (Environments, Secrets and Configuration)?" "large_header" true) }}
```

File: themes/default/layouts/product/esc.html [hunk 5]
```diff
@@ -204,36 +204,36 @@ <h2>Frequently asked questions</h2>
...
                 <div class="accordion-item-body-no-animation text-base">
                     <p>
-                        Pulumi ESC (Environments, Secrets and Configurations), is an open source project and managed service as part of Pulumi Cloud that enables teams to manage hierarchical collections of configuration and secrets, and then consume those configuration and secrets from a variety of different infrastructure and application services.
+                        Pulumi ESC (Environments, Secrets and Configuration), is an [open source project](https://github.com/pulumi/esc) and managed service as part of Pulumi Cloud that enables teams to manage hierarchical collections of configuration and secrets, and then consume those configuration and secrets from a variety of different infrastructure and application services.
```
Comment (cnunciato) [inline themes/default/layouts/product/esc.html:210]: Never "those configuration":

```suggestion
                        Pulumi ESC (Environments, Secrets and Configuration), is an [open source project](https://github.com/pulumi/esc) and managed service of Pulumi Cloud that enables teams to manage hierarchical collections of configuration and secrets and consume them from a variety of different infrastructure and application services.
```

File: themes/default/layouts/product/esc.html [hunk 6]
```diff
@@ -204,36 +204,36 @@ <h2>Frequently asked questions</h2>
...
                     </p>
                     <p>
-                        {{ "Pulumi ESC integrates with Pulumi Cloud identity and RBAC to provide rich control over access to secret configuration within an organization.  Pulumi ESC supports multiple Configuration Providers, enabling static key/value configuration, as well as dynamically retrieved configuration and secrets via OIDC and additional providers like 1Password and Vault.  Pulumi ESC is available via the new `esc` CLI, Pulumi Cloud, the Pulumi Cloud REST API, and Pulumi IaC stack configuration." | markdownify }}
+                        {{ "Pulumi ESC integrates with Pulumi Cloud identity and RBAC to provide rich control over access to secret configuration within an organization. Pulumi ESC supports multiple Configuration Providers, enabling static key/value configuration, as well as dynamically retrieved configuration and secrets via OIDC and additional providers like 1Password and Vault.  Pulumi ESC is available via the new `esc` CLI, Pulumi Cloud, the Pulumi Cloud REST API, and Pulumi IaC stack configuration." | markdownify }}
```
Comment (cnunciato) [inline themes/default/layouts/product/esc.html:213]: Not a proper noun:

```suggestion
                        {{ "Pulumi ESC integrates with Pulumi Cloud identity and RBAC to provide rich control over access to secret configuration within an organization. Pulumi ESC supports multiple configuration providers, enabling static key/value configuration as well as dynamically retrieved configuration and secrets via OIDC and additional providers like 1Password and Vault.  Pulumi ESC is available via the new `esc` CLI, Pulumi Cloud, the Pulumi Cloud REST API, and Pulumi IaC stack configuration." | markdownify }}
```

File: themes/default/layouts/product/esc.html [hunk 7]
```diff
@@ -204,36 +204,36 @@ <h2>Frequently asked questions</h2>
...
+                        An environment describes a collection of secrets and configuration values. It is typically intended to capture the configuration needed to work with a particular environment - for example the production environment for your key customer or line of business service.
                     </p>
-                    <p>An environment is represented by a YAML document.  This document has two top level entries:</p>
+                    <p>An environment is represented by a YAML document. This document has two top level entries:</p>
                     <p class="ml-8">
-                        {{ "`imports`:  An optional set of other environments that this environment derives from, enabling composition of environments, and avoiding repetition across environments." | markdownify}}
+                        {{ "`imports`: An optional set of other environments that this environment derives from, enabling composition of environments, and avoiding repetition across environments." | markdownify}}
```
Comment (cnunciato) [inline themes/default/layouts/product/esc.html:225]:
```suggestion
                        {{ "`imports`: An optional set of other environments that this environment derives from, enabling composition of environments and avoiding repetition across environments." | markdownify}}
```

File: themes/default/layouts/product/esc.html [hunk 8]
```diff
@@ -204,36 +204,36 @@ <h2>Frequently asked questions</h2>
...
                     </p>
                     <p class="ml-8">
                         {{ "`values`: An arbitrary nested collection of key/value pairs representing top level configuration values. " | markdownify}}
                     </p>
                 </div>
             </li>
             <li class="accordion-item text-2xl py-3">
-                {{ partial "accordian-header" (dict "text" "What secrets and configurations can I manage with ESC?" "large_header" true) }}
+                {{ partial "accordian-header" (dict "text" "What secrets and configuration can I manage with ESC?" "large_header" true) }}
```
Comment (cnunciato) [inline themes/default/layouts/product/esc.html:233]:
```suggestion
                {{ partial "accordian-header" (dict "text" "What secrets and configuration can I manage with Pulumi ESC?" "large_header" true) }}
```

File: themes/default/layouts/product/esc.html [hunk 9]
```diff
@@ -204,36 +204,36 @@ <h2>Frequently asked questions</h2>
...
                 <div class="accordion-item-body-no-animation text-base">
                     <p>
-                        Pulumi ESC enables teams to manage hierarchical collections of configuration and secrets (‘“environments”), and then consume those configuration and secrets from a variety of different infrastructure and application services. Examples include the management of AWS configurations and secrets, API keys, database credentials, or environment-specific variables such as service endpoints and AWS regions.
+                        Pulumi ESC enables teams to manage hierarchical collections of configuration and secrets (“environments”), and then consume those configuration and secrets from a variety of different infrastructure and application services. Examples include the management of AWS configurations and secrets, API keys, database credentials, or environment-specific variables such as service endpoints and AWS regions.
```
Comment (cnunciato) [inline themes/default/layouts/product/esc.html:236]:
```suggestion
                        Pulumi ESC enables teams to manage hierarchical collections of configuration and secrets (“environments”) and consume them from a variety of different infrastructure and application services. Examples include the management of AWS configurations and secrets, API keys, database credentials, and environment-specific variables such as service endpoints and AWS regions.
```

**Review by cnunciato — APPROVED** [review]
Lots of little things to fix up, but leaving an approval to unblock. 🚀

**Pull request merged**